frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -124,48 +131,93 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
-          buttonClassName="btn-primary"
-          title="Create New"
-          items={ownedCRDs.reduce((acc, crd) => ({...acc, [crd.name]: crd.displayName}), {})}
-          onChange={(name) => history.push(route(name))} /> }
-        { status.phase === ClusterServiceVersionPhase.CSVPhaseSucceeded && ownedCRDs.length === 1 && <Link to={route(ownedCRDs[0].name)} className="btn btn-primary">{`Create ${ownedCRDs[0].displayName}`}</Link> }
+  return <React.Fragment>
+    <ScrollToTopOnMount />
+
+    <div className="co-m-pane__body">
+      <SectionHeading text="ClusterServiceVersion Overview" />
+      <div className="co-m-pane__body-group">
+        <div className="row">
+          <div className="col-sm-6">
+            <div style={{marginBottom: '15px'}}>
+              { status.phase !== ClusterServiceVersionPhase.CSVPhaseSucceeded && <button disabled={true} className="btn btn-primary">Create New</button> }
+              { status.phase === ClusterServiceVersionPhase.CSVPhaseSucceeded && ownedCRDs.length > 1 && <Dropdown
+                buttonClassName="btn-primary"
+                title="Create New"
+                items={ownedCRDs.reduce((acc, crd) => ({...acc, [crd.name]: crd.displayName}), {})}
+                onChange={(name) => history.push(route(name))} /> }
+              { status.phase === ClusterServiceVersionPhase.CSVPhaseSucceeded && ownedCRDs.length === 1 && <Link to={route(ownedCRDs[0].name)} className="btn btn-primary">{`Create ${ownedCRDs[0].displayName}`}</Link> }
+            </div>
+            <ResourceSummary resource={props.obj} showNodeSelector={false} showPodSelector={false} />
+          </div>
+          <div className="col-sm-6">
+            <React.Fragment>
+              {status.phase !== ClusterServiceVersionPhase.CSVPhaseSucceeded
+                ? <React.Fragment>
+                  <dt>Status</dt>
+                  <dd className="co-clusterserviceversion-detail__error-box">
+                    <strong>{status.phase}</strong>: {status.message}
+                  </dd>
+                </React.Fragment>
+                : <React.Fragment>
+                  <dt>Status</dt>
+                  <dd>{status.phase}</dd>
+                  <dt>Status Reason</dt>
+                  <dd>{status.message}</dd>
+                </React.Fragment>}
+              <dt>Provided APIs</dt>
+              {spec.customresourcedefinitions.owned.map(desc => <dd key={desc.name} style={{display: 'flex', alignItems: 'center'}}>
+                <ResourceIcon kind={referenceForCRDDesc(desc)} /><strong>{desc.displayName}</strong>&nbsp;&nbsp;<span className="text-muted">({desc.name})</span>&nbsp;- {desc.description}
+              </dd>)}
+              <dt>Operator Deployments</dt>
+              {spec.install.spec.deployments.map(({name}, i) => <dd key={i}><ResourceLink name={name} kind="Deployment" namespace={metadata.namespace} /></dd>)}
```

Comment:
Is `name` / `namespace` unique so that it can be used for `key`?

Reply:
All of these deployments would be in the same namespace, so Kube will guarantee they have unique names.

Reply:
OK, yeah, that seems like a better key than index